ui/lib/replication/addon/templates/mode.hbs

```diff
@@ -1,57 +1,62 @@
 <section class="section">
   <div class="container is-widescreen">
-    {{#if model.replicationAttrs.replicationEnabled}}
-      <PageHeader as |p|>
-        <p.top>
-          <nav class="key-value-header breadcrumb">
+    {{#if (and (eq model.drMode "secondary") (eq model.drModeInit "primary"))}}
+      <nav class="navbar"></nav>
```

Comment:
This one doesn't have the `page-container` wrapping div and the one above does. Should we make them consistent?

Reply:
Good call.  I think the solution is to remove the page-container from above.  I'll play around and amend.  Thanks!